packages/grid/_modules_/grid/components/toolbar/DensitySelector.tsx

```diff
@@ -65,7 +65,7 @@ export function DensitySelector() {
   };
 
   // Disable the button if the corresponding is disabled
-  if (options.disableColumnFilter) {
+  if (options.disableDensitySelector) {
```

Comment:
separate issue?

Reply:
Yes turns out I missed that one. I'll open a separate PR now.

Reply:
Great, we could rebase since it's merged

Reply:
Done